mkosi/archive.py

```diff
@@ -103,8 +103,8 @@ def make_cpio(
     sandbox: SandboxProtocol = nosandbox,
 ) -> None:
     if not files:
-        files = src.rglob("*")
-    files = sorted(files)
+        with chdir(src):
+            files = sorted(Path(".").rglob("*"))
```

Comment:
If files doesn't come in sorted already this the files in the cpio might not be sorted anymore and the whole thing less reproducible.

Reply:
Yes but the only user of does does make sure the files are already sorted

Reply:
True, though it is something to keep in mind in addition. The overhead of sorting again should be negligible.